### ISSUE TITLE
When Rectangle is opened it will crash Chrome Dev.

### ISSUE THREAD

**Issue by marcosriani** [rxhanson/Rectangle#1145]
- [ ] This is a new bug that hasn't been documented in a closed issue or in the Readme.

When Rectangle is opened it will crash Chrome Dev, I can open Chrome Dev but if I try to click and open a new tab or click on the menu of the Browser, it will automatically close it. This same behaviour does not happen on regular Chrome, on on Chrome Dev.

macOS version: Mac os Ventura  13.3.1
Rectangle version: 0.68
Logs if applicable (In Rectangle menu, hold option, "View Logging..."):

**Comment by rxhanson** [comment]
Is this still an issue? If so, this is likely something that would have to be addressed on the Chrome Dev side.

**Comment by kevinhendricks** [comment]
FYI, Sigil is a cross platform Qt 6.5.2 app (open source ebook editor) and one of our macOS user's reported crashes related to Rectangle.app and Sigil-2.0.1.

The backtraces show a qt filedialog causing an abort after it receives some Accessibility Events but only when Rectangle is present.  I am going to build Rectangle and try to track down what is conflicting.

Have you ever had any other reported issues with Chrome, Electron, or Qt apps?

Any idea about how Rectangle would cause issues with a QFileDialog in the latest LTS Qt 6.5.2 based app?

**Comment by rxhanson** [comment]
@kevinhendricks thanks for investigating. Cross platform apps, particularly Electron ones tend to have hiccups other apps using the Accessibility API more than any other kind of app. Rectangle will listen to click events for the drag to snap functionality, and this is typically what causes issues. It's worth noting that Rectangle isn't doing anything strange with the click listening, and that all 3rd party window management apps that have drag to snap functionality will use the same API, and that there is no way to achieve this functionality without doing it this way. Rectangle hasn't changed the way this functionality operates since it was created.

**Comment by kevinhendricks** [comment]
Okay, Qt devs worked on this issue and tracked down and fixed bugs in their Accessibility code and in their QListView implementation.

See https://bugreports.qt.io/browse/QTBUG-114423

I hosted a link to a patch at the end of that issue.  I downloaded their patch, rebuilt Qt 6.5.2 with it,  I used that patched Qt 6.5.2 to rebuild Sigil and loaded up Rectangle.app and no more crashes.

That Qt bug fix should become part of their next Qt 6.5.X release and be part of dev and future Qt 6.6 releases.
Thought you might like to know.
